scripts/bootstrap.sh

```diff
@@ -66,7 +67,8 @@ install_direnv() {
 }
 
 direnv_shell_integration() {
-	DIRENV_BIN="/usr/local/bin/direnv"
+	DIRENV_BIN="$(command -v direnv)"
+	DIRENV_BIN="${DIRENV_BIN:-$HOME/.nix-profile/bin/direnv}"
```

Comment:
Do we want a default value here? Or would it be better just to error if direnv isn't found?

Reply:
I have an edge case when `direnv` installed using `asdf` and have not been active in the current directory, I haven't fully worked out a way to detect this scenario so I just set a default value to avoid the script error.

I would like to address this in the next around of bugfix but right now I want SSL fix out ASAP